aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSyncRequestFactory.java

```diff
@@ -440,29 +444,42 @@ private static Object appSyncOpValue(QueryOperator<?> qOp) throws DataStoreExcep
     private static Object extractAssociateId(ModelField modelField, Model instance, ModelSchema schema)
             throws AmplifyException {
         final Object fieldValue = extractFieldValue(modelField.getName(), instance, schema);
+
         if (modelField.isModel() && fieldValue instanceof Model) {
             return ((Model) fieldValue).getId();
         } else if (modelField.isModel() && fieldValue instanceof Map) {
             return ((Map<?, ?>) fieldValue).get("id");
         } else {
+            LOG.info("AppSyncRequestFactory extractAssociatedId field value: " + fieldValue);
+            LOG.info("AppSyncRequestFactory extractAssociatedId Model field: " + modelField);
+            LOG.info("AppSyncRequestFactory extractAssociatedId instance: " + instance);
+            LOG.info("AppSyncRequestFactory extractAssociatedId instance: " + schema);
```

Comment:
```suggestion
            LOG.info("AppSyncRequestFactory extractAssociatedId schema: " + schema);
```

Reply:
These logs have been removed from the last commit. Sorry about the confusion.